.swiftpm/xcode/xcshareddata/xcschemes/PriorityQueueModule.xcscheme

```diff
@@ -1,77 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why was this file deleted?

Reply:
I am not sure how that file got deleted as I never touched that folder. Next time I'll double check which files have changed before submitting a PR so this doesn't happen again.

Reply:
No worries! Do you know how to revert the deletion?

Reply:
Yes! I used this command format `git checkout HEAD~1 file.txt` to bring back the deleted file.

Reply:
Oh, nice! I didn't realize you could use `git checkout HEAD~1` on specific files! (I guess that makes sense — I use `git checkout --` on specific files all the time.) 👍  #TIL